apps/docs/src/pages/docs/_meta.ts

```diff
@@ -6,6 +6,7 @@ export default {
   introduction: "Introduction",
   "get-started": "Get Started",
   changelog: "Changelog",
+  status: "Status",
```

Comment:
can we align the sidebar & page titles. We call it "Status" here, but "Project Status" on the page - it feels like something's missing